extractors/register_parser.py

```
@@ -2,9 +2,160 @@
+from dotenv import load_dotenv
+
 
 from datetime import datetime
 
+# ✅ 환경 변수 및 Vision API 클라이언트 설정
+_vision_client = None
+
+
+def get_vision_client():
+    global _vision_clinet
+    if _vision_client is None:
```

Ruff (0.12.2)

Local variable `_vision_client` referenced before assignment

(F823)